app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+
+    storage = {}
+
+    def wrapper(*args) -> Callable:
```

Comment:
what if function take key arguments?
```suggestion
    def wrapper(*args, **kwargs) -> Ant:
```